github/activity_events.go

```diff
@@ -28,9 +28,9 @@ func (e Event) String() string {
 	return Stringify(e)
 }
 
-// Payload returns the parsed event payload. For recognized event types,
+// ParsePayload parses the event payload. For recognized event types,
 // a value of the corresponding struct type will be returned.
-func (e *Event) Payload() (payload interface{}) {
+func (e *Event) ParsePayload() (payload interface{}, err error) {
```

Comment:
I think option 2 is a great idea, @shurcooL. The fewer projects we break, the better, and this gives people the opportunity to update their code at their convenience.  I like it. Thanks.